tests/unit/test_routes.py

```diff
@@ -27,8 +28,17 @@ def test_index_route(test_client):  # pylint: disable=redefined-outer-name
 
     This test sends a GET request to the root URL ("/") using the test client
     and verifies that the response has a status code of 200 and contains
-    the expected content "Hello, World!" in the response data.
+    the expected content "CI migration process" in the response data.
     """
     response = test_client.get("/")
```

Comment:
Could we test other methods here, like what is POST returning, since methods like POST, PUT and PATCH will also return `200` for some atm.